proxy/update_proxy.sh

```diff
@@ -49,9 +54,21 @@ function changelog_update {
     rm debian.changelog-bk
 }
 changelog_update $VERSION
-sed -e "s/@VERSION@/$VERSION/" cc-proxy.spec-template > cc-proxy.spec
-sed -e "s/@VERSION_DEB_TRANSFORM@/$VERSION_DEB_TRANSFORM/g;" -e "s/@HASH_TAG@/$short_hashtag/g;" cc-proxy.dsc-template > cc-proxy.dsc
-sed -e "s/@VERSION_DEB_TRANSFORM@/$VERSION_DEB_TRANSFORM/g;" -e "s/@HASH_TAG@/$short_hashtag/g;" debian.control-template > debian.control
+
+REVISION=$(($(cat release) + 1))
```

Comment:
nitpick please change `@REVISION@` by `@RELEASE@`

Reply:
oh! specfiles call it release, debian dsc files call it revision. Sometimes I mix the things.